### ISSUE TITLE
Power supply monitoring

### ISSUE THREAD

**Issue by dyvyak** [mrRobit/Zabbix_templates#10]
Can you please add power supply monitoring to template?
Thx

**Comment by mrRobit** [comment]
Yes I will add it. But please note, power supply monitoring capability via SNMP for PA-3400 series will be added in the PAN-OS 11.1

**Comment by mrRobit** [comment]
@dyvyak what firewall model and PAN-OS do you want to monitor the power supplies on?

**Comment by dyvyak** [comment]
Hi, 3410 and 1420 with panos  11.0.3-h5
Thanks

________________________________
